docs/integration/OpenAPI.md

```diff
@@ -74,4 +74,4 @@ components {
 
 You can enable logging invocations in the runtime by setting logger
 `pl.touk.nussknacker.openapi.enrichers.[enricher name]` to `DEBUG`. In particular, setting `pl.touk.nussknacker.openapi.enrichers` 
-level to `DEBUG` will turn on logging on all enrichers. Please remember to configure logging on appropriate component (e.g. Lite runtime or Flink TaskManager).
\ No newline at end of file
+level to `DEBUG` will turn on logging on all enrichers. Please remember to configure logging on appropriate component (e.g. Lite runtime or Flink TaskManager).
```

Comment:
Term "component" has a special meaning in Nu case, so I would avoid using it here. Do we mean engine? Additionally, instead of "pls remember", I would prefer some explanation which would enforce mental model of what is going on here. Example:

_For enriched level logging to work, engine  level logging has to be enabled_  (links to documentation or page titles or ... ) . 

I would also have this sentence at the beginning of this section (?), because it is a _prerequisite_, so adding this information at the end is not optimal to me.

Reply:
done